packages/teams-js/src/internal/telemetry.ts

```diff
@@ -1,5 +1,17 @@
 import { debug as registerLogger, Debugger } from 'debug';
 
+import { UUID } from './uuidObject';
+
+// Each teamsjs instance gets a unique identifier that will be prepended to every log statement
+export const teamsJsInstanceIdentifier = new UUID();
+
+// Every log statement will get prepended with the teamsJsInstanceIdentifier and a timestamp
+const origFormatArgs = registerLogger.formatArgs;
+registerLogger.formatArgs = function (args) {
+  args[0] = `(${new Date().toISOString()}): ${args[0]} [${teamsJsInstanceIdentifier.toString()}]`;
```

Comment:
How often is this called? Just the first time the library initializes, or every time we log? Wondering if we should be concerned about the perf of `new Date()`. For some reason I have a memory that it is not performant? But I could definitely be misremembering.

Reply:
I did some light searching online and couldn't find anything to confirm my memory. Maybe it's fine?

Reply:
Per in-person discussion, logs are not written frequently enough for us to worry about performance right now, but it will be something we keep an eye on.